coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/CoordinatorRuntime.java

```diff
@@ -911,6 +912,24 @@ public void run() {
             }
         }
 
+        /**
+         * Completes the given event once all pending writes are completed. 
+         *
+         * @param event             The event that must be completed when the 
+         *                          records are written.
```

Comment:
Could we also update the param description?
```suggestion
         * @param event The event to complete once all pending writes are completed.
```